src/components/Contacts/ContactDetails/ContactDetailsHeader/ContactDetailsHeader.test.tsx

```diff
@@ -18,11 +18,11 @@ describe('ContactDetails', () => {
       </GqlMockedProvider>,
     );
 
-    await waitFor(() => expect(getByText('loading')).toBeInTheDocument());
+    expect((await findAllByRole('ContactName'))[0]).toBeInTheDocument();
```

Comment:
Sorta seems like a loading state should be able to find this in the document...

Reply:
Hmm yeah now that I think about this, this is not a good test for the loading state.  When it's loading, almost all the content should be `Skeleton` components.  Should I search for items with the Skeleton type to tell if it is a loading state?

Reply:
I think that sounds ok. Or at least assert that the data isn't in the document. But that doesn't test much... What you've got looks good :)